addonscript/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.black]
+line-length = 160
+target-version = ["py37", "py38"]
+include = '\.(wsgi|pyi?)$'
+exclude = '''
+/(
+    \.eggs
+  | \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | \.cache
+  | \.cache_py3
+  | _build
+  | buck-out
+  | build
+  | dist
+  | ui
+)/
+'''
+
+[tool.isort]
+line_length = 160
+known_first_party = 'addonscript'
```

Comment:
here I also had not_skip `__init__.py` which I found useful when its not just a blank file.

Of course I'm not a huge fan of the parenthesis and is why I also disabled that but I'm not too worried about it